pkg/kubelet/dockertools/docker.go

```diff
@@ -117,29 +117,22 @@ type dockerContainerCommandRunner struct {
 }
 
 // The first version of docker that supports exec natively is 1.3.0 == API 1.15
-var dockerAPIVersionWithExec = []uint{1, 15}
+var dockerAPIVersionWithExec, _ = docker.NewAPIVersion("1.15")
 
 // Returns the major and minor version numbers of docker server.
-func (d *dockerContainerCommandRunner) GetDockerServerVersion() ([]uint, error) {
+func (d *dockerContainerCommandRunner) GetDockerServerVersion() (docker.APIVersion, error) {
 	env, err := d.client.Version()
 	if err != nil {
 		return nil, fmt.Errorf("failed to get docker server version - %v", err)
 	}
-	version := []uint{}
-	for _, entry := range *env {
-		if strings.Contains(strings.ToLower(entry), "apiversion") || strings.Contains(strings.ToLower(entry), "api version") {
-			elems := strings.Split(strings.Split(entry, "=")[1], ".")
-			for _, elem := range elems {
-				val, err := strconv.ParseUint(elem, 10, 32)
-				if err != nil {
-					return nil, fmt.Errorf("failed to parse docker server version %q: %v", entry, err)
-				}
-				version = append(version, uint(val))
-			}
-			return version, nil
-		}
+
+	apiVersion := env.Get("ApiVersion")
```

Comment:
We also report docker server version as part of GetDockerServerVersion defined at https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L1095

That one only includes docker server version here. I am ok to change to APIVersion, but we need to fix that output though.